package-examples/helloworld-tshirt/README.md

```diff
@@ -0,0 +1,75 @@
+# helloworld-tshirt
+
+This is a package that has both a setter and a simple hydration function.  The
+hydration function examines the configuration and looks for the t-shirt size
+value.  Depending on the size (small, medium, large) it then sets the 
+resource sizes.
+
+## Steps
+
+This is a sample workflow where 
```

Comment:
Remove this line.

Reply:
Sounds good.  Removing this in other files as well to have uniformity.